lpa-submissions-e2e-tests/cypress/integration/pageFooter/pageFooter.step.js

```diff
@@ -0,0 +1,18 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { clickFooterLink } from '../../support/common/clickFooterLink';
```

Comment:
clickFooterLink is unlikely to be used outside of this context. Why not just have the function directly where we use it to save the cognitive load of imports?

Reply:
updated as per review comments